modules/light-clients/08-wasm-light-clients/contracts/ethereum-light-client/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub mod contract;
```

Comment:
I think we should enforce documentation in all public API in this crate like we do in ibc-go (since people and future devs might want to read). We can also add stricter linting as follows:

```suggestion
#![deny(clippy::nursery, clippy::pedantic, warnings, missing_docs)]

pub mod contract;
```

Reply:
Sure! I will do that in a separate task: #7604